### PR TITLE
refactor: refactor `get_table_names` and `get_columns`

### PR DESCRIPTION
**Pull request by HuaHuaY**
1. Refactor `get_table_names`. Use `information_tables` instead of `show tables`.
2. Refactor `get_columns`. I think this PR will help us to add types' arguments in the future.
3. Correct the version to be consistent with the version previously released on PyPI.

### CODE REVIEW

File: sqlalchemy_risingwave/base.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "varchar": sqltypes.VARCHAR,  # DataType::Varchar
     "time without time zone": sqltypes.Time,  # DataType::Time
     "timestamp without time zone": sqltypes.TIMESTAMP,  # DataType::Timestamp
-    "timestamp with time zone": sqltypes.TIMESTAMP(True),  # DataType::Timestampz
```
Comment (BowenXiao1999) [inline sqlalchemy_risingwave/base.py:23]: Why change this arguments? We do not need True or False for the timestamp type?

File: sqlalchemy_risingwave/__init__.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from sqlalchemy.dialects import registry as _registry
 
-__version__ = "0.0.3"
```
Comment (BowenXiao1999) [inline sqlalchemy_risingwave/__init__.py:3]: Is it true that we already reach 0.0.7?

**Review by BowenXiao1999 — COMMENTED** [review]
Rest LGTM, good job!

File: sqlalchemy_risingwave/base.py (hunk 1 above)
Comment (HuaHuaY) [reply sqlalchemy_risingwave/base.py:23]: It's moved to line 93. So we can store only type, no need to store object in `_type_map`.

File: sqlalchemy_risingwave/__init__.py (hunk 2 above)
Comment (HuaHuaY) [reply sqlalchemy_risingwave/__init__.py:3]: Emm, I push a version 0.0.7 to pypi to override 0.0.6. We have talked about the version in slack. So I change this version number.

**Review by BowenXiao1999 — APPROVED** [review]

**Pull request merged**